lib/ProductOpener/Display.pm

```diff
@@ -6253,6 +6253,16 @@ HTML
 
 	}
 
+	# GS1-Prefixes for restricted circulation numbers within a company - warn for possible conflicts
+	if ($code =~ /^(?:(?:0{7}[0-9]{5,6})|(?:04[0-9]{10,11})|(?:[02][0-9]{2}[0-9]{5}))$/) {
```

Comment:
It's a non-capturing group, so that it is used to group the expressions, but doesn't try to allocate a variable for it for output. Just saves a bit of memory.